sig-gen-util/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "signatures"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Can you add a publish = false line here? See https://github.com/FuelLabs/sway/blob/acc6c438a67c42cc913744523d2b4f3544434e13/test/Cargo.toml#L6

Reply:
Done.